src/mesa/drivers/dri/Android.mk

```diff
@@ -35,6 +35,7 @@ MESA_DRI_C_INCLUDES := \
 	$(addprefix $(MESA_TOP)/, $(mesa_dri_common_INCLUDES)) \
 	$(MESA_TOP)/src/gallium/include \
 	$(MESA_TOP)/src/gallium/auxiliary \
+	system/core/include \
```

Comment:
Do we still need this line? Android.common.mk has include this line. Please check if this makefile has include Android.common.mk.